server/src/resolvers.ts

```diff
@@ -35,6 +35,62 @@ export const resolvers: ResolverMap = {
 
       await user.save();
       return true;
+    },
+    createTournament: async (
+      _,
+      { userId, name, description, startDate, amountTeams }
```

Comment:
You can do { userId, ...others } and then const tournament = Tournament.create(others)